config/settings.yml

```diff
@@ -19,62 +19,62 @@
       master:
     manageiq-v2v:
       master:
-  :productization_name: manageiq
+  :productization_name: xaasio-cmp
   :remotes:
-    :upstream: git@github.com:ManageIQ
+    :upstream: https://gitlab.xaasio.com/xaasio-cmp
```

Comment:
The mirror section below not be changed because this is the source of the mirroring - if you need to change the destination of the mirroring, you can add in a `downstream:` key here and that will be used as the destination for the mirror.  productization_name will be used as the repo prefix for the mirroring.